src/yup.test.ts

```diff
@@ -119,26 +120,33 @@ describe('yupResolver', () => {
         createdOn: null,
         foo: [{ loose: null }],
       };
-      const resolve = await yupResolver(schema)(data, {}, true);
-      expect(resolve).toMatchSnapshot();
-      expect(resolve.errors['foo'][0]['loose']).toBeDefined();
-      expect(resolve.errors['foo'][0]['loose'].types).toMatchInlineSnapshot(`
+
+      const output = await yupResolver(schema)(
+        // @ts-expect-error
```

Comment:
minor (don't have to fix it): Is this due to reuse the same schema? maybe worth to use a new schema instead?

Reply:
This is normal because `yupResolver` have a [better type support now :) ](https://github.com/react-hook-form/resolvers/pull/92/files/8701c7062718187ce7bbe1693a2d1a4105f216c9#diff-c1efacb5b2a76e9e4b9c1fd06e62bcac926439ffc153d798f5e440e79f3a2882)

The schema is good but passed data are intentionally false, that's why TS throw an error.